### ISSUE TITLE
API proposal: Update Archive Layout Class for streams

### ISSUE THREAD

**Issue by normanargueta** [opentok/Opentok-Python-SDK#140]
This is a proposal to add a new OpenTok Python SDK method to change layout classes for an OpenTok stream.

### opentok.set_stream_class_list(session_id, stream_class_list)

The layout classes define how the stream is displayed in the layout of a composed OpenTok archive. For more information, see [docs](https://tokbox.com/developer/rest/#change-stream-layout-classes-composed).

### Usage

``` python
# This list contains the information of the streams that will be updated. Each element
# in the list is a dictionary with two properties: 'id' and 'layoutClassList'. The 'id'
# property is the stream ID (a String), and the 'layoutClassList' is an array of class
# names (Strings) to apply to the stream.
payload = [
    {'id': '7b09ec3c-26f9-43d7-8197-f608f13d4fb6', 'layoutClassList': ['focus']},
    {'id': '567bc941-6ea0-4c69-97fc-70a740b68976', 'layoutClassList': ['top']},
    {'id': '307dc941-0450-4c09-975c-705740d08970', 'layoutClassList': ['bottom']}
]

opentok.set_stream_class_list('SESSIONID', payload)
```

**Comment by msach22** [comment]
@normanargueta I think we already have the `set_archive_layout` method: https://github.com/opentok/Opentok-Python-SDK/blob/master/opentok/opentok.py#L601

This is for setting the stream class list layout so I think we should call it something like this: `set_stream_class_lists` or `set_streamClass_lists`

**Comment by normanargueta** [comment]
@msach22 you are right, I updated the method name

**Comment by msach22** [comment]
Released with v2.9.0
